openshift/manifests/18-deployment-openshift-operator-controller-operator-controller-controller-manager.yml

```diff
@@ -81,6 +81,9 @@ spec:
             - mountPath: /etc/containers
               name: etc-containers
               readOnly: true
+            - mountPath: /etc/operator-controller
+              name: auth-from-host
```

Comment:
```suggestion
            - mountPath: /etc/operator-controller/auth.json
              readOnly: true
              name: global-auth-file
```

Reply:
This is a generated file, so the source will have to be modified.

Reply:
ah yes! i'll suggest in the right spot.

Reply:
Moved my suggestion to the source of this change.